### ISSUE TITLE
Terms & names

### ISSUE THREAD

**Issue by Veldkip** [XFox111/TabsAsideExtension#79]
When I use the feature it's a whole window that's put aside right?  Or is this called a tab? Or tabpage? And all the webpages are then called tabsheets?  You are using the term 'collection'.......
What I'm already missing for as long as I'm working with more than 1 window/tab is the possibility to give that a name.
Could me that's possible on collection level and that I just missed it...

**Comment by XFox111** [comment]
Hi, thank you for your feedback!
Could you please tell me more about the issue? I didn't quite get it

**Comment by Veldkip** [comment]
I cannot make it more clear!!!!
My time is pretty valuable.........
Are you developer? Or administration?

Raul Veldhuizen       [image: 📱] +31654 378 678

Lepelaar 7
1191SL Ouderkerk
[image: ☎] +3120 496 5557




<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virusvrij.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

Op do 27 mei 2021 om 16:09 schreef Eugene Fox ***@***.***>:

**Comment by XFox111** [comment]
> When I use the feature it's a whole window that's put aside right? Or is this called a tab? Or tabpage? And all the webpages are then called tabsheets? You are using the term 'collection'.......

When you click "Set current tabs aside" button, it saves all opened tabs in your browser window into a collection (or a set). Then it closes all opened tabs in the window

> I cannot make it more clear!!!!

Well, then I'm afraid there's nothing else I can help you with

> My time is pretty valuable.........

Everybody's time is

> Are you developer? Or administration?

I'm creator/developer/maintainer/administrator of this project. It's my personal one

If don't have any other questions, I'd like to close the issue

**Comment by Veldkip** [comment]
I always have several windows open, with different subjects. Cannot name these windows. Would be good if I could when using Tabs aside.
Will not better the situation when re-opening the collection again but OK, it makes them more recognisable.
I noticed re-opening happens in the current window. I would have preferred 'in a new window'.
Setting aside will take less memory then the 'sleep'function I trust?

**Comment by XFox111** [comment]
Hi, sorry for such delayed response.

So, from what you've written, I can suggest these features to make:
- Add a "Close the window after setting tabs aside" toggle to settings
- Add an option "Open in a new window" to sets' three-dot menu
- Add a "Close the panel after restoring tabs" toggle to settings
Is that what would you like?

> Setting aside will take less memory then the 'sleep'function I trust?

Not sure about memory improvements, but I don't think it will break something

**Comment by Veldkip** [comment]
Yes, these additions would be helpful to me. Of course always hard to tell how widespread my way of doing things is....
The sleeping function has bettered my situation as far as the limit my computer can handle, aka the amount of crashes. No change in the ease of finding the window the tab I want is in.

**Comment by XFox111** [comment]
Hi, @Veldkip! Have you got my response? I saw your reply in the mail, but it seems it's been deleted from GitHub

**Comment by Veldkip** [comment]
terribly busy!
will get back on it later.
sorry.

Raul Veldhuizen       [image: 📱] +31654 378 678

Lepelaar 7
1191SL Ouderkerk
[image: ☎] +3120 496 5557




<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virusvrij.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

Op zo 1 aug. 2021 om 10:33 schreef Eugene Fox ***@***.***>:

**Comment by AmineI** [comment]
It is quite difficult to understand you. I'll try to answer anyway.
As far as I know, tabs are set aside into a "collection", which is a set of multiple tabs. You can rename collections, if that's the question. You just have to click on their name 

> I cannot make it more clear!!!!
My time is pretty valuable.........

Everyone's time is as well. If you are the only one able to understand your question, then you will be the only one able to find your answer. 

> I always have several windows open, with different subjects. Cannot name these windows. 

On Microsoft Edge you can do that, right click on the window and choose "name window" 
![image](https://user-images.githubusercontent.com/15179425/129030451-1ebb78b4-8747-4721-90c5-74d20ef7fec3.png)

> Setting aside will take less memory then the 'sleep' function I trust?

Yes. Microsoft Edge's "Sleeping tabs" feature still uses some memory to keep the tab & data available, whereas tabs set aside use 
no memory but take some time to reopen.
